bin/templates/cordova/lib/builders/GradleBuilder.js

```diff
@@ -211,3 +209,46 @@ module.exports = GradleBuilder;
 function isAutoGenerated(file) {
     return fs.existsSync(file) && fs.readFileSync(file, 'utf8').indexOf(MARKER) > 0;
 }
+
+/**
+ * A special superspawn-like implementation, required to workaround the issue
```

Comment:
Not even an additional option, but an ability to accept any valid `stdio` option - `superspawn` now knows only `default`, `ignore` and `inherit`. I agree, this will be the most correct solution, but will require patching this in cordova-common and releasing it before next android release.  

@jasongin, what do you think?